File: 006/2023_11_16/abc174b.jl
N, D = broadcast(n -> parse(Int, n), split(readline()))

answer = 0

for i in 1:N
    X, Y = broadcast(n -> parse(Int, n), split(readline()))

    if X * X + Y * Y ≤ D * D
        global answer += 1
    end
end

println(answer)

File: 01/003/2023_10_29/abc174b.py
N, D = map(int, input().split())
xyN = []

answer = 0

for i in range(0, N):
    xyN.append(list(map(int, input().split())))

for xy in xyN:
    if xy[0] * xy[0] + xy[1] * xy[1] <= D * D:
        answer += 1

print(answer)
